pkg/urbit/vere/io/term.c

```diff
@@ -526,10 +494,12 @@ static void
 _term_it_show_more(u3_utty* uty_u)
 {
   if ( c3y == u3_Host.ops_u.tem ) {
-    _term_it_write_str(uty_u, "\n");
-  } else {
-    _term_it_write_str(uty_u, "\r\n");
+    _term_it_dump(uty_u, TERM_LIT("\n"));
+  }
+  else {
+    _term_it_dump(uty_u, TERM_LIT("\r\n"));
```

Comment:
Might be a good time to ask: what's the deal with the "terminal assumptions" flag, and why does it influence how we print newlines?

Reply:
`u3_Host.ops_u.tem` (the `-t` flag) disables "terminal assumptions": specifically, the assumption that we're connected to an interactive tty. Setting that option means that we just append output, and avoid refreshing the prompt, ringing the bell, &c. We may want to disable style-rendering entirely as well.

Reply:
Would I be correct in interpreting that as "logging mode" in a sense? Disabling style seems the right thing there, yes.

Reply:
Yeah, that's exactly right. It's "non-interactive foreground mode", while "daemon mode" is "non-interactive background mode".